Source/GSDisplayServer.m

```diff
@@ -601,6 +601,12 @@ - (void) miniwindow: (int) win
   [self subclassResponsibility: _cmd];
 }
 
+/** Hides all application windows */
+- (void) hidewindow: (int) win
+{
+  [self subclassResponsibility: _cmd];
```

Comment:
I think it is better to leave this method empty. You only provided an implementation for the x11 backend on Windows this will result in an exception and hiding no longer will work there.

Reply:
Also hidewindow: has a boolean return value. This should be respected and NO returned.

Reply:
Fixed.